interpreter/runtests.py

```diff
@@ -30,6 +31,8 @@
 
 class RunTests(unittest.TestCase):
   def _runCommand(self, command, logPath, expectedExitCode = 0):
+    if isWindows:
+      command = command.replace('\'', '"').replace('\\', '/')
```

Comment:
Yeah the underlying problem is that the interpreter doesn't recognize `\` in paths, but I wanted to make a quick fix for the time being, I might look into this later on.